.github/workflows/Dockerfile.focal

```diff
@@ -6,13 +6,17 @@ COPY tests/python/requirements.txt /tmp/
 RUN apt-get update && \
     apt-get install -y --no-install-recommends apt-utils ca-certificates gnupg wget pkg-config software-properties-common && \
     wget -qO /etc/apt/trusted.gpg.d/vkpartner.asc https://artifactory-external.vkpartner.ru/artifactory/api/gpg/key/public && \
+    wget -qO - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add - && \
+    add-apt-repository ppa:ubuntu-toolchain-r/test && \
+    echo "deb http://apt.llvm.org/focal/ llvm-toolchain-focal-18 main" >> /etc/apt/sources.list && \
+    echo "deb-src http://apt.llvm.org/focal/ llvm-toolchain-focal-18 main" >> /etc/apt/sources.list && \
```

Comment:
Are we sure we need the source code?

Reply:
what do you mean? we are not building llvm from source here. https://apt.llvm.org/ says this is the way

Reply:
also `ppa:ubuntu-toolchain-r` needed to install g++-11, which is first version of gcc which supports coroutins in c++20 standard without `-fcoroutins` flag

Reply:
ok